docs/src/js/ds6/main.js

```diff
@@ -2,36 +2,31 @@
 
 // makeup-js accessibility modules
 var Expander = require('makeup-expander');
-var keyEmitter = require('makeup-key-emitter');
 var scrollKeyPreventer = require('makeup-prevent-scroll-keys');
 var modal = require('makeup-modal');
 var transition = require("./transition");
 var FloatingLabel = require('makeup-floating-label');
 
-// util function
+// util functions
 function nodeListToArray(nodeList) {
     return Array.prototype.slice.call(nodeList);
 }
 
-// BUTTON WIDGET
-
-// simple button logic on buttons
-nodeListToArray(document.querySelectorAll('.btn:not([aria-disabled="true"]):not(.dialog-button):not(.tooltip__close)')).forEach(function(el, i) {
-    el.addEventListener('click', function(e) {
-        alert('You clicked ' + this);
-    });
-});
+function querySelectorAllToArray(selector, parentNode) {
```

Comment:
I don't see any use of `parentNode` ever being passed into this function. If so, this could be simplified to just use `document` all the time.

Unless you foresee it the param being used. I'm just not sure. It has a pretty specific use case right now.